test/functional/database-schema/column-types/postgres/column-types-postgres.ts

```diff
@@ -4,18 +4,38 @@ import {Connection} from "../../../../../src/connection/Connection";
 import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from "../../../../utils/test-utils";
 import {PostWithoutTypes} from "./entity/PostWithoutTypes";
 import {Post} from "./entity/Post";
+import {PostgresDriver} from "../../../../../src/driver/postgres/PostgresDriver";
 
 describe("database schema > column types > postgres", () => {
+    // Save the original set timezone for later restore
+    let originalTimezone: string;
```

Comment:
do we really need to "restore it"? Isn't `SET TIME ZONE` per-session configuration?

Reply:
I thought it is set until restart of postgres

Reply:
It is unnecessary. I removed it.